sous/Starter.php

```diff
@@ -20,7 +20,8 @@ public static function installTheme() {
   // New DrupalFinder to get the Composer root path.
   $drupalFinder = new DrupalFinder();
   $drupalFinder->locateRoot(getcwd());
-  $composerRoot = str_replace('-', '_', strtolower(basename($drupalFinder->getComposerRoot())));
+  $removeChars = array("-", "_", " ");
+  $composerRoot = str_replace($removeChars, '', strtolower(basename($drupalFinder->getComposerRoot())));
   // Execute the Emulsify theme build based on composer create path.
   shell_exec ("cd web/themes/contrib/emulsify-design-system/ && php emulsify.php $composerRoot");
```

Comment:
The following lines should also change from `web/themes/contrib/emulsify-design-system/` to `web/themes/contrib/emulsify-drupal`. When running the composer-create command I get `sh: line 0: cd: web/themes/contrib/emulsify-design-system/: No such file or directory` error.